library/Stratosphere/Types.hs

```diff
@@ -1,14 +1,19 @@
-{-# LANGUAGE DeriveAnyClass #-}
-{-# LANGUAGE DeriveGeneric #-}
+{-# LANGUAGE DeriveAnyClass    #-}
```

Comment:
Yeah that was stylish Haskell. I use it on my own projects but have it set to be a manual operation. Spaced out about this project not using that style elsewhere. Fixed.